<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
    <line>
      <![CDATA[open Application]]>
    </line>
    <line>
      <![CDATA[sample]]>
    </line>
    <line>
      <![CDATA[Data:Username]]>
    </line>
    <line>
      <![CDATA[Row: 1]]>
    </line>
    <line>
      <![CDATA[Test Method: testA is FAILED & photo is ./photo]]>
    </line>
  </reporter-output>
  <suite name="Default suite" duration-ms="32487" started-at="2018-09-27T09:11:39Z" finished-at="2018-09-27T09:12:12Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="32487" started-at="2018-09-27T09:11:39Z" finished-at="2018-09-27T09:12:12Z">
      <class name="script.TestDemo">
        <test-method status="PASS" signature="openApplication()[pri:0, instance:script.TestDemo@371a67ec]" name="openApplication" is-config="true" duration-ms="24881" started-at="2018-09-27T09:11:39Z" finished-at="2018-09-27T09:12:04Z">
          <reporter-output>
            <line>
              <![CDATA[open Application]]>
            </line>
          </reporter-output>
        </test-method> <!-- openApplication -->
        <test-method status="FAIL" signature="testA()[pri:0, instance:script.TestDemo@371a67ec]" name="testA" duration-ms="4983" started-at="2018-09-27T09:12:04Z" finished-at="2018-09-27T09:12:09Z">
          <exception class="java.lang.AssertionError">
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError
at org.junit.Assert.fail(Assert.java:86)
at org.junit.Assert.fail(Assert.java:95)
at script.TestDemo.testA(TestDemo.java:22)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[sample]]>
            </line>
            <line>
              <![CDATA[Data:Username]]>
            </line>
            <line>
              <![CDATA[Row: 1]]>
            </line>
          </reporter-output>
        </test-method> <!-- testA -->
        <test-method status="PASS" signature="closeApplication(org.testng.ITestResult)[pri:0, instance:script.TestDemo@371a67ec]" name="closeApplication" is-config="true" duration-ms="2496" started-at="2018-09-27T09:12:09Z" finished-at="2018-09-27T09:12:12Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=testA status=FAILURE method=TestDemo.testA()[pri:0, instance:script.TestDemo@371a67ec] output=sample]]]>
              </value>
            </param>
          </params>
          <reporter-output>
            <line>
              <![CDATA[Test Method: testA is FAILED & photo is ./photo]]>
            </line>
          </reporter-output>
        </test-method> <!-- closeApplication -->
      </class> <!-- script.TestDemo -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
